project_template/webpack.config.js

```diff
@@ -36,7 +36,7 @@ const plugins = [
 
 const optimization = {
   splitChunks: {
```

Comment:
@pascalw upon further exploration its best to disable chunking when not in production, as it causes some issues with Karma and unnecessary delays in test execution: `splitChunks: isProd && { chunks: "all" }`.

Issue I witnessed (with chunking on) was Karma silently failing to load tests that import code that makes use of the `h` pragma for some reason. Also saw webpack producing chunks for the Karma entrypoint which is totally unnecessary.

Reply:
Ah gotcha. This is probably why we had this chunks related config in Karma config before.